log4j-core/src/main/java/org/apache/logging/log4j/core/config/LoggerConfig.java

```diff
@@ -214,7 +214,7 @@ public Filter getFilter() {
             return filter;
         }
 
-        public B withtFilter(Filter filter) {
+        public B withFilter(Filter filter) {
```

Comment:
I have mixed feelings about this change: yes, it would be nice to have a correctly spelled method, but the other one was already released and it is API.

Don't change the name of the method, deprecate it and add another one with the correct spelling.

Reply:
I deprecated the misspelled method and created a new one with the correct spelling as suggested.

Reply:
There were two such cases.
What is the rule for naming methods?
When should the name be `setFilter` and when `withFilter`?
I just noticed that the appender was changed from  `setFilter` and when `withFilter`

Reply:
> I have mixed feelings about this change: yes, it would be nice to have a correctly spelled method, but the other one was already released and it is API.
> 
> Don't change the name of the method, deprecate it and add another one with the correct spelling.

Just add a new method with the correct name and deprecate the bogus one. Simple ;-)